sql/catalyst/src/test/scala/org/apache/spark/sql/connector/InMemoryTable.scala

```diff
@@ -46,10 +49,15 @@ class InMemoryTable(
   private val allowUnsupportedTransforms =
     properties.getOrDefault("allow-unsupported-transforms", "false").toBoolean
```

Comment:
Shall we remove line 46 and 47 since this is not used after this PR?

Reply:
I considered it, but we may still want to test creating a table with a different transform that is not implemented. Since we already support it, I thought it wasn't worth removing this. I can if you feel strongly about it.

Reply:
Got it. Then, no problem.